lib/ecto/multi.ex

```diff
@@ -65,7 +65,7 @@ defmodule Ecto.Multi do
           # under keys we used for naming the operations.
         {:error, failed_operation, failed_value, changes_so_far} ->
           # One of the operations failed. We can access the operation's failure
-          # value (like changeset for operations on changesets) to prepare
+          # value (like changeset for operations on changesets) to prepare a
           # proper response. We also get access to results of any operations
           # that succeeded before the indicated operation failed.
```

Comment:
I was also curious if it would be worth mentioning that the successful operations were rolled back? This makes it seem like maybe it wasn't run in a transaction, for the DBs that support transactions obviously.

For DBs like Mongo, I suspect these would be the successful operations that actually did succeed and the developer would be responsible for cleaning up at this point.